DESCRIPTION

```diff
@@ -6,7 +6,8 @@ Authors@R:
            comment = c(ORCID = "0000-0003-0862-2018"))
 Description: Standalone shims for functions from packages such as {cli}, {forcats}, {stringr}, etc.
 License: MIT + file LICENSE
-URL: http://www.danieldsjoberg.com/standalone/
+URL: http://www.danieldsjoberg.com/standalone/,
```

Comment:
Ah, yes! We did need the website there. We can delete my URL, since it's no longer associated with this repo.

But we also need to make the website work again. IDK what happened, but it's not there anymore.

Reply:
I'm not sure where the error is coming from - when I build the site locally, it renders correctly :/
I've also checked all the links to make sure they work, I'll keep investigating